_posts/2024-02-21-replacing-default-admin-credentials.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: post
+title:  "Enhancing Security in OpenSearch 2.12.0: The End of Default Admin Password"
+authors:
+ - dchanp
+date: 2024-02-21
+categories:
+ - community-updates
+ - technical-posts
+meta_keywords: "OpenSearch Security, Security Demo Configuration"
+meta_description: Learn about the changes to Security plugin's demo configuration setup
+has_math: false
+has_science_table: false
+---
+
+In the realm of data management and search engines, OpenSearch has been a key player, offering robust features and a flexible platform for users worldwide. With the release of OpenSearch 2.12.0, a significant security upgrade is set to transform how administrators initially set up their clusters using the security plugin's demo configuration tool.
+
+**The Vulnerability of Default Passwords**
+
+Until OpenSearch version 2.11.x, the security plugin shipped with a default admin password, 'admin'. While this facilitated ease of use during initial setup, it posed a significant security risk. Despite warnings from the demo configuration tool against using this default password in production, the risk of leaving clusters vulnerable was a pressing concern.
```

Comment:
I would explicitly say "demo" ...the security plugin shipped with a default admin password **in the demo configuration file**... (or something like that)